analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Nullable.cs

```diff
@@ -85,4 +85,18 @@ public void Nullable_HasValue_ReadsBoolConstraintFromObjectConstraint()
         validator.ValidateTag("HasValueAfterNull", x => x.HasConstraint(BoolConstraint.False).Should().BeTrue());
         validator.ValidateTag("SymbolAfterNull", x => x.HasConstraint(ObjectConstraint.Null).Should().BeTrue());
     }
+
+    [TestMethod]
+    public void Nullable_Ctor_NoArguments_SetsNullConstraint()
+    {
+        const string code = """
+            int? explicitType = new Nullable<int>();
+            int? targetTyped = new();
```

Comment:
Propably later:
```cs
    void M<T>() where T: struct
    {
        T? a = new T(); // NotNull (this should come from me)
        T? b = new T?(); // Null  (this should come from you)
    }
```

Reply:
This already works - so I've added those cases